src/sentry/api/serializers/models/project.py

```diff
@@ -176,6 +176,7 @@ def format_options(attrs: defaultdict(dict)):
             "sentry:performance_issue_creation_rate"
         ),
         "filters:blacklisted_ips": "\n".join(options.get("sentry:blacklisted_ips", [])),
+        "filters:react-hydration-errors": bool(options.get("filters:react-hydration-errors", True)),
```

Comment:
@jan-auer testing locally, even if I send "True" this option never gets updated. do you know why? maybe I am missing something